File: Android/app/src/main/java/com/hhp227/datemate/data/model/Post.kt
package com.hhp227.datemate.data.model

import com.google.firebase.Timestamp

data class Post(
    var uid: String = "",
    var userId: String = "",
    var title: String = "",
    var content: String = "",
    var imageUrls: List<String> = emptyList(),
    var likeCount: Int = 0,
    var createdAt: Timestamp? = null
)

File: iOS/DateMate/Data/Model/Post.swift
//
//  Post.swift
//  DateMate
//
//  Created by 홍희표 on 2021/12/11.
//

import Foundation
import FirebaseFirestore

struct Post: Identifiable {
    var id: String
    
    var userId: String
    
    var title: String
    
    var content: String
    
    var imageUrls: [String]
    
    var likeCount: Int
    
    var createdAt: Timestamp?
}
